<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qnn</class>
 <widget class="QWidget" name="qnn">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QNeuralNetwork</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QPushButton" name="startTraining">
       <property name="text">
        <string>Start Training</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stopTraining">
       <property name="text">
        <string>Stop Training</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="PlotWidget" name="accuracyGraph" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="networkName"/>
     </item>
     <item>
      <widget class="QPushButton" name="loadNetwork">
       <property name="text">
        <string>Load Network</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveNetwork">
       <property name="text">
        <string>Save Network</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
